config/prod/bridgeserver2.yaml

```diff
@@ -13,11 +13,7 @@ parameters:
   AwsSnsBounceNotificationEndpoint: bridge-bounce-notifications@sagebase.org
   AwsSnsNotificationEndpoint: bridgeops@sagebase.org
   BootstrapAdminEmail: !ssm /bridgeserver2-prod/BootstrapAdminEmail
-  BootstrapAdminPassword: !ssm /bridgeserver2-prod/BootstrapAdminPassword
-  BootstrapApiEmail: !ssm /bridgeserver2-prod/BootstrapApiEmail
-  BootstrapApiPassword: !ssm /bridgeserver2-prod/BootstrapApiPassword
-  BootstrapSharedEmail: !ssm /bridgeserver2-prod/BootstrapSharedEmail
-  BootstrapSharedPassword: !ssm /bridgeserver2-prod/BootstrapSharedPassword
+  BootstrapAdminSynapseUserId: !ssm /bridgeserver2-prod/BootstrapAdminSynapseUserId
```

Comment:
I don't think the Synapse User ID needs to go into SSM. We could just make it a string literal in this file.